ragaai_catalyst/tracers/agentic_tracing/tracers/main_tracer.py

```
@@ -42,19 +42,18 @@
 )
 
 from ....ragaai_catalyst import RagaAICatalyst
-from ragaai_catalyst.tracers.upload_traces import UploadTraces
 
 
 class AgenticTracing(
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)